lib/buildsystems/cmake.rb

```diff
@@ -1,16 +1,10 @@
 require 'package'
 
 class CMake < Package
-  def self.cmake_options(options = '')
-    return (@cmake_options = options if options)
-  end
-
-  def self.check?(bool = true)
-    return (@check = bool)
-  end
+  property :cmake_options
+  boolean_property :run_tests
```

Comment:
Would it make sense to define the `run_tests` property in package.rb since it's used generically with all build systems?

Reply:
I thought it'd be better to keep it in each individual buildsystem, since it's not used by non-buildsystem packages. Fine with either, though.

Reply:
It just follows the DRY (don't repeat yourself) principle.

Reply:
Fair enough. I'll update the PR when I get the chance.